spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/RateLimitAutoConfiguration.java

```diff
@@ -86,8 +86,7 @@ public RateLimiter consultRateLimiter(final ConsulClient consulClient, final Obj
     }
 
     @ConditionalOnMissingBean(RateLimiter.class)
-    @ConditionalOnMissingClass({"org.springframework.data.redis.core.RedisTemplate",
-            "com.ecwid.consul.v1.ConsulClient"})
+    @ConditionalOnMissingClass({ClassFullName.RedisTemplate, ClassFullName.ConsulClient})
```

Comment:
Maybe this should be replaced with repository property?

Reply:
fair point, I'll update the PR shortly